doc/api.yml

```diff
@@ -2757,6 +2968,64 @@ components:
         creation_datetime:
           type: string
           format: date-time
+    CreateEvent:
+      type: object
+      properties:
+        id:
+          type: string
+        description:
+          type: string
+        begin:
+          type: string
+          format: date-time
+        end:
+          type: string
+          format: date-time
+        event_type:
+          $ref: '#/components/schemas/EventType'
+        planner_id:
+          description: user id
+          type: string
+        course_id:
+          type: string
+        groups:
+          type: array
+          items:
+            $ref: '#/components/schemas/GroupIdentifier'
+    EventParticipantStatus:
+      type: string
+      enum:
+        - MISSING
+        - PRESENT
+    EventParticipantUser:
+      allOf:
+        - $ref: '#/components/schemas/UserIdentifier'
+      type: object
+      properties:
+        event_status:
+          $ref: '#/components/schemas/EventParticipantStatus'
+    Event:
+      type: object
+      properties:
+        id:
+          type: string
+        type:
+          $ref: '#/components/schemas/EventType'
+        description:
+          type: string
+        planner:
+          $ref: '#/components/schemas/UserIdentifier'
+        course:
+          $ref: '#/components/schemas/Course'
+    EventParticipant:
+       type: object
+       properties:
+         group:
+           $ref: '#/components/schemas/GroupIdentifier'
+         students:
+           type: array
+           items:
+             $ref: '#/components/schemas/EventParticipantUser'
```

Comment:
rename this EventParticipant to GroupedEventParticipant, it will only serve as rest response, not as a payload